srcpkgs/sof-firmware/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'sof-firmware'
 pkgname=sof-firmware
-version=1.5.1
+version=1.6rc3
 revision=1
 archs="i686* x86_64*"
-wrksrc=sof-bin-stable-v${version}
+wrksrc="sof-bin-1.6-rc3"
```

Comment:
The `1.6-rc3` version number should be a variable, because it seems to be used in multiple places. You could do something like:

```
version=1.6rc3
_major=${version%rc*}
_minor=${version#${_major}}
```

And replace where it's used.